SonarProjectConfig.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<SonarProjectConfig xmlns="http://www.sonarsource.com/msbuild/analyzer/2021/1">
```

Comment:
`SonarProjectConfig` - same as file name to be unique, even when it has dedicated xmlns